.ci/setup.sh

```diff
@@ -25,13 +26,10 @@ else  # Linux
         echo 'debconf debconf/frontend select Noninteractive' | sudo debconf-set-selections
 
         sudo apt-get update
-        sudo apt-get install -y --no-install-recommends \
+        sudo apt-get install --no-install-recommends -y \
```

Comment:
For my own understanding...does the order of these arguments change the behavior of this call? Or are you just taking this opportunity to standardize all the `apt-get install` calls?

I'm totally ok with this change being part of this PR, just want to understand.

Reply:
I'm 
> taking this opportunity to standardize all the `apt-get install` calls?

🙂